pipeline/02-assess.R

```diff
@@ -268,7 +296,8 @@ assessment_data_pin <- assessment_data_merged %>%
     meta_year, meta_pin, meta_pin10, meta_triad_code, meta_township_code,
     meta_nbhd_code, meta_tax_code, meta_class, meta_tieback_key_pin,
     meta_tieback_proration_rate, meta_cdu, meta_modeling_group,
```

Comment:
Is there any documentation that needs to be updated with new values in pin / card output files?

Reply:
Nope! This should just get automatically added to the equivalent tables in Athena once it's crawled.